web/services/store/index.js

```diff
@@ -0,0 +1,50 @@
+import { createStore, combineReducers, compose } from 'redux';
+import { reactReduxFirebase, firebaseReducer } from 'react-redux-firebase';
+import firebase from 'firebase';
+
+const prodConfig = {
+  // TODO: setup production firestore project
+};
+
+const devConfig = {
+  apiKey: 'AIzaSyCBkQHeikIsiYZ2yOHiqH_mGJKDWMDU500',
+  authDomain: 'nwhacks-2019.firebaseapp.com',
+  databaseURL: 'https://nwhacks-2019.firebaseio.com',
+  projectId: 'nwhacks-2019',
+  storageBucket: 'nwhacks-2019.appspot.com',
+  messagingSenderId: '98283589440',
+};
+
+const config = {
+  production: prodConfig,
```

Comment:
can probably just declare the configurations in-line here 🤷‍♂️

Reply:
i don't like nested objects 😞